discord/webhook.py

```diff
@@ -642,6 +652,62 @@ def edit(self, **kwargs):
 
         return self._adapter.edit_webhook(**payload)
 
+    async def _async_move(self, channel_id, token, payload):
+        await self._adapter.move_webhook(token, **payload)
+        self.channel_id = channel_id
+
+    def _sync_move(self, channel_id, token, payload):
+        self._adapter.move_webhook(token, **payload)
+        self.channel_id = channel_id
+
+    def move_to(self, channel):
+        """|maybecoro|
+
+        Moves this Webhook.
+
+        If the webhook is constructed with a :class:`RequestsWebhookAdapter` then this is
+        not a coroutine.
+
+        .. warning::
+
+            This will only work if you fetch the webhook from a client, as there is a state attached.
+
+        .. versionadded:: 1.2.0
+
+        Parameters
+        -----------
+        channel: :class:`TextChannel`
+            Where to move the webhook. Must be a text channel.
```

Comment:
No need to say "Must be a text channel." because the line just above says TextChannel.